ui/src/app/pages/data/cohort/attributes-page.spec.tsx

```diff
@@ -193,29 +188,33 @@ describe('AttributesPageV2', () => {
   });
 
   it('should render a single input for EQUAL operator and disable calculate button when empty', async () => {
-    const wrapper = component();
-    const numericalDropdown = getNumericalDropdown(wrapper, '0');
-    numericalDropdown.props.onChange({
-      originalEvent: undefined,
-      value: Operator.EQUAL,
-      target: { id: '', name: '', value: Operator.EQUAL },
-      stopPropagation: () => {},
-      preventDefault: () => {},
-    });
-    await waitOneTickAndUpdate(wrapper);
-    expect(wrapper.find('[id="numerical-input-0-0"]').length).toBe(2);
-    expect(
-      wrapper.find('[data-test-id="attributes-calculate-btn"]').first().props()
-        .disabled
-    ).toBeTruthy();
-    const numericalInput = getNumericalInput(wrapper, '0');
-    numericalInput.simulate('change', { target: { value: 100 } });
-    await waitOneTickAndUpdate(wrapper);
-    expect(
-      wrapper.find('[data-test-id="attributes-calculate-btn"]').first().props()
-        .disabled
-    ).toBeFalsy();
-    wrapper.find('[data-test-id="attributes-calculate-btn"]').simulate('click');
-    expect(mockCountParticipants).toHaveBeenCalledTimes(1);
+    const user = userEvent.setup();
+    await component();
+
+    screen.getByDisplayValue(/any value/i);
+    // Simulate the dropdown change
+    const dropdown = screen.getByRole('button', { name: /select operator/i });
+    user.click(dropdown);
+    const equalOption = await screen.findByText('Equals');
+    user.click(equalOption);
+    await screen.findByDisplayValue('Equals');
```

Comment:
is this needed?  Does this find the `equalOption` that we just clicked, or something else?

Reply:
It is a little redundant.

This ensures that the text box is updated to the value we just clicked in the dropdown. 

The line below it ensures that a textbox with the old value no longer exists.

Do you have a preference on what to do here?